ngi_pipeline/engines/qc_ngi/workflows.py

```diff
@@ -202,6 +202,9 @@ def fastq_to_be_analysed(fastq_files, analysis_dir, output_footers):
         if not m:
             # fastq file name doesn't match expected pattern -- let be serious.. we do NOT process it
             continue
+        if '_I1_' in fastq_file:
```

Comment:
Hi @Hammarn so this makes the job, it needs to be tested as you never know with ngi-pipeline.
Couple of comments:
  
 - make the check a bit more "locked", there might be unfortunate cases where a sample contains "_I1_" in the name (especially Uppsala samples). So use a more "complex" regex to be sure to capture the index fastq file only
 - I assume there might be the case of `_I2_` also, so maybe worth including it. This can be done also addressing the previous point.
 - as I said this works, however it would made sense to send to fastqc also the index sequence, to check the quality of the index, this might be interesting in 10X experiments. But this requires much more changes to the code.

Reply:
Thanks for the input. Yeah I was thinking the same thing, probably should be a bit more robust and you never know with ngi_pipeline. 
I'll come back to this in the coming week.